### PR TITLE
fix beforeCapture hook mapping

### PR DESCRIPTION
**Pull request by SophieDePaula**
[Trello card](https://trello.com/c/vcJjdJxY/1300-cypress-beforecapturescreenshot-hook-not-working)
The mapping for `beforeCaptureScreenshot` hook was incorrect.

### CODE REVIEW

File: pending-changes.yaml [hunk 1]
```diff
@@ -111,6 +111,7 @@ Apollo SDK:
       `extractText` now supports regions that don't use hints while using `x`/`y` coordinates
     - add 'waitBeforeCapture' as an available type for 'Eyes.Open' and 'Eyes.Check' (for Typescript)
     - Support data urls in iframes
+    - fix beforeCapture hook not being honored
```
Comment (SophieDePaula) [inline pending-changes.yaml:114]: @benbabayoff8 fyi

Comment (amitzur) [reply pending-changes.yaml:114]: 1. beforeCaptureScreenshot is the correct term.
2. honored --> respected

**Review by amitzur — APPROVED** [review]

**Comment by amitzur** [comment]
@SophieDePaula actually, I don't think these should be specifix tests. There should be a generic test for this

File: pending-changes.yaml [hunk 2]
```diff
@@ -107,7 +107,7 @@ Apollo SDK:
   feature:
     - 
   bug-fix:
-    - 
+    - fix beforeCaptureScreenshot hook not being respected
```
Comment (SophieDePaula) [inline pending-changes.yaml:110]: @benbabayoff8 fyi

**Pull request merged**